canonical-websites/docs.ubuntu.com
# Add initial Dockerfile

- Add Dockerfile for building web app image

Fixes #87 

QA
--

Build and run the docker image:

``` bash
docker build --tag docs.ubuntu.com --build-arg COMMIT_ID=`git rev-parse HEAD` .
docker run --rm -p 80:80 docs.ubuntu.com
```

Go to <http://127.0.0.1> and see the docs site
